ten-leetcode-problems/ValidParantheses.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ten_leetcode_problems
+{
+    internal static class ValidParantheses
+    {
+        public static bool IsValidParentheses(string s)
+        {
+            Stack<char> stack = new Stack<char>();
+
+            foreach (char c in s)
+            {
+
+                if (c == '(' || c == '[' || c == '{')
```

Comment:
lets avoid hardcoding these chars in the code, and maybe have some dictionary where we save each opening bracket with it's ending bracket.